packages/components/src/public/QueryModel/withQueryModels.tsx

```diff
@@ -56,6 +56,55 @@ export function withSearchParams<T>(Component: WithSearchParamsComponent<T>): Co
     return Wrapped;
 }
 
+function columnHasFilter(fieldKey, filters: Filter.IFilter[]): boolean {
+    return filters.some(filter => filter.getColumnName().toLowerCase() === fieldKey);
```

Comment:
should we just do `fieldKye.toLowerCase()` here so we don't have to rely on the caller doing it?

Reply:
I was doing it in the caller above so it's not called during every iteration of some, but I'll just lowerCase it in this method before calling some